app/models/citysdk/note.rb

```diff
@@ -0,0 +1,22 @@
+# frozen_string_literal: true
+
+module Citysdk
+  class Note < ::Comment
+    include Citysdk::Serialization
+
+    self.serialization_attributes = %i[id jurisdiction_id comment datetime service_request_id author]
+    alias_attribute :service_request_id, :issue_id
+    alias_attribute :comment, :message
+    alias_attribute :datetime, :created_at
+
+    def jurisdiction_id; end
+
+    def author
+      user.to_s
+    end
+
+    def author=(value)
+      self.user = User.find_by(email: value)
```

Comment:
```suggestion
      self.user = User.active.find_by(email: value)
```
Wäre es hier nicht besser nur die aktiven Nutzer zu berücksichtigen?